sh/tests/solutions/joker-num.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+if [[ $# != 1
+    || -z "$1"
```

Comment:
Nice touch! Even if it's not needed to pass this exercise, could be nice to add this to the `hints` part of the subject

Reply:
done